### ISSUE TITLE
Website: tabs example is broken for keyboard users

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#523]
This is some technical debt we created when stripping away all the jQuery. Now that the JavaScript situation is much cleaner and saner (same JS running on both ds4 and ds6 sites), we can start adding back in some of the basic keyboard functionality. This one should be pretty easy using `makeup-roving-tabindex`.

**Comment by ianmcburnie** [comment]
Fixed in 7.0.0 branch.
